### DISCUSSION TITLE
Changes to existing records not saving, although twill reports saved without issue

### DISCUSSION THREAD

**Discussion by adam-jones-net** [area17/twill#1970] [Q&A]
I've seen this before but surprised to not be able to find a discussion here on github about it.
Can someone give me some obvious things to check ? 

This is an up to date twill install that has been running some years without issue.

**Comment by kallefrombosnia** [comment]
Do you use any overriding on Module Repository?
I've had the same issue which drove me mad and after long debugging, I noticed that I forgot to call `parent::afterSave()` as final on my method override. This can happen to every other method that Twill uses.

Also, check if your Model has fillable properties which you use.

You can also join our Discord server so that we can provide help live. https://discord.link/twill

**Reply by adam-jones-net** [reply]
@kallefrombosnia  thanks for reply. No I don't believe I am doing either of these for this specific module.       What I do see is that the problem doesn't occur on my dev setup (WIMP) but does on the live server (LAMP).
